sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/service/query/executor/DocumentPropertyComparator.java

```diff
@@ -1,12 +1,20 @@
 package io.stargate.sgv2.docsapi.service.query.executor;
 
+import com.google.protobuf.ByteString;
```

Comment:
Not necessarily ideal to have this dependency (since this is not protobuf related?), but I guess it's fine as we already have this dependency for Bridge access.

Reply:
Hmm but the bridge model data does expose this for bytes transfer.. Am I getting something wrong?